File: Dante1/Module1.06/Zadanie6.20/main.c
#include <stdio.h>

int main()
{
    int matrixA[5][5];
    int vectorA[5] = {0};

    int matrixB[5][5];
    int vectorB[5] = {0};

    printf("Enter:\n");
    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < 5; j++)
        {
            int check = scanf("%i",&matrixA[i][j]);
            if(check != 1)
            {
                printf("Incorrect input\n");
                return 1;
            }
        }
    }
    printf("Enter:\n");
    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < 5; j++)
        {
            int check = scanf("%i",&matrixB[i][j]);
            if(check != 1)
            {
                printf("Incorrect input\n");
                return 1;
            }
        }
    }

    for(int i = 0; i < 5; i++)
    {
        int sum1 = 0, sum2 = 0;
        for(int j = 0; j < 5; j++)
        {
            sum1 = sum1 + matrixA[j][i];
            sum2 = sum2 + matrixB[j][i];
        }
        vectorA[i] = sum1;
        vectorB[i] = sum2;
    }

    printf("\n");
    for (int i = 0; i < 5; i++)
    {
        for (int j = 0; j < 5; j++)
        {
            if (vectorA[j] >= vectorB[j])
            {
                printf("%i ", matrixA[i][j]);
            }
            else
            {
                printf("%i ", matrixB[i][j]);
            }
        }
        printf("\n");
    }


    return 0;
}

File: Dante1/Module1.06/Zadanie6.20/main.rs
fn main() {
    let mut input: String = String::new();

    println!("Enter");
    let mut matrixA: [[i32; 5]; 5] = [[0; 5]; 5];
    let mut vectorA: [i32; 5] = [0; 5];
    for i in 0..5 {
        for j in 0..5 {
            std::io::stdin().read_line(&mut input).expect("Error");
            matrixA[i][j] = input.trim().parse().expect("err");

            input.clear();
        }
    }

    println!("Enter");
    let mut matrixB: [[i32; 5]; 5] = [[0; 5]; 5];
    let mut vectorB: [i32; 5] = [0; 5];
    for i in 0..5 {
        for j in 0..5 {
            std::io::stdin().read_line(&mut input).expect("Error");
            matrixB[i][j] = input.trim().parse().expect("err");

            input.clear();
        }
    }

    for i in 0..5 {
        let mut sum1: i32 = 0;
        let mut sum2: i32 = 0;
        for j in 0..5 {
            sum1 = sum1 + matrixA[j][i];
            sum2 = sum2 + matrixB[j][i];
        }
        vectorA[i] = sum1;
        vectorB[i] = sum2;
    }

    for i in 0..5 {
        for j in 0..5 {
            if vectorA[j] >= vectorB[j]
            {
                println!("{} ", matrixA[i][j]);
            }
            else
            {
                println!("{} ", matrixB[i][j]);
            }
        }
        println!();
    }

}
